docs/sources/flow/reference/components/mimir.rules.kubernetes.md

```diff
@@ -120,11 +120,11 @@ The `match_expression` block describes a Kubernetes label match expression for r
 
 The following arguments are supported:
 
-Name       | Type           | Description                                        | Default | Required
------------|----------------|----------------------------------------------------|---------|---------
-`key`      | `string`       | The label name to match against.                   |         | yes
-`operator` | `string`       | The operator to use when matching.                 |         | yes
-`values`   | `list(string)` | The values used when matching.                     |         | no
+Name       | Type       | Description                                        | Default | Required
+-----------|------------|----------------------------------------------------|---------|---------
+`key`      | `string`   | The label name to match against.                   |         | yes
+`operator` | `string`   | The operator to use when matching. |         | yes
+`values`   | `[]string` | The values used when matching.                     |         | no
```

Comment:
Did we want to revert this change as well?

Suggestion in case you want to apply from GH:
```suggestion
Name       | Type           | Description                                        | Default | Required
-----------|----------------|----------------------------------------------------|---------|---------
`key`      | `string`       | The label name to match against.                   |         | yes
`operator` | `string`       | The operator to use when matching.                 |         | yes
`values`   | `list(string)` | The values used when matching.                     |         | no
```